geo/Cargo.toml

```diff
@@ -28,7 +28,7 @@ __allow_deprecated_features = []
 earcutr = { version = "0.4.2", optional = true }
 spade = { version = "2.10.0", optional = true }
 float_next_after = "1.0.0"
-geo-types = { version = "0.7.17", features = ["approx", "rstar_0_12"] }
+geo-types = { version = "0.7.18", features = ["approx", "rstar_0_12"] }
```

Comment:
We've got to release `geo-types` before we can release `geo`, which references the new feature flags.